activejob/lib/active_job/queue_adapters/inline_adapter.rb

```diff
@@ -0,0 +1,15 @@
+module ActiveJob
+  module QueueAdapters
+    class InlineAdapter
+      class << self
+        def enqueue(job, *args)
+          job.new.execute *args
+        end
+
+        def enqueue_at(*)
+          raise NotImplementedError.new("Use a queueing backend to enqueue jobs in the future. Read more at https://github.com/rails/activejob")
```

Comment:
@seuros I was wondering if `enqueue_at` could just delegate to `enqueue` ignoring the timeout settings. 

My motivation is because in development, I would like to change only the adapter from `shoryuken` to `inline`, but because of `NotImplementedError`, I can't. I would need to create my own `Inline` or add `ifs` in the code.

If delegating from `enqueue_at` to `enqueue` is too ✨ , we could also log a warning message saying it's ignoring the timeout options.

WDYT?